init/postgres/test_sample.sql

```diff
@@ -100,7 +100,15 @@ CREATE TABLE test_sample.all_types
 
     -- enum
     mood_ptr test_sample.mood,
-    mood test_sample.mood NOT NULL DEFAULT 'ok'
+    mood test_sample.mood NOT NULL DEFAULT 'ok',
+
+    -- range types
```

Comment:
`postgres` and `cockroachdb` share the same test code. If you add new columns to postgres db,  you'll have to add new columns to `cockroach` db as well. Unfortunately `cockroachdb` doesn't support range types at the moment. 
So I think the best would be to create a new table just for range types, and make range type test run only for postgres. You can use `skipForCockroachDB(t)` function.